### ISSUE TITLE
odf->pdf converter does not mantains border padding

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#195]
```
border padding is ignored for paragraph, see example 
ParagraphTextWithBorder.odt.
```

Original issue reported on code.google.com by `abe...@gmail.com` on 14 Nov 2012 at 8:39

**Comment by GoogleCodeExporter** [comment]
```
Problem fixed - see example in ParagraphTextWithBorder.odt
@Angelo
In open offiece borders may be joined for adjacent paragraphs. That's why I 
added a new class StylableParagraphWrapper which is responsible for paragraph 
border, padding and background.
I don't know if border joining is possible in docx. If not, existing 
StylableParagraph with added padding will be sufficient.
```

Original comment by `abe...@gmail.com` on 3 Jun 2013 at 10:44

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.0.3
```

Original comment by `abe...@gmail.com` on 3 Jun 2013 at 10:48
- Changed state: **Fixed**
